File: 2525.py
class Solution:
    def categorizeBox(self, length: int, width: int, height: int, mass: int) -> str:
        volume = length * width * height
        bulky, heavy = False, False

        if(length >= 10**4) or (width >= 10**4) or (height >= 10**4) or (volume >= 10**9):
            bulky = True
        if(mass >= 100):
            heavy = True
        if(bulky and heavy):
            return "Both"
        elif(bulky):
            return "Bulky"
        elif(heavy):
            return "Heavy"
        else:
            return "Neither"

File: 2525.cpp
class Solution {
public:
    string categorizeBox(int length, int width, int height, int mass) {
        bool bulky = false, heavy = false;
        long long int volume = (long) length * width * height;

        if(length >= 10000 || width >= 10000 || height >= 10000 ||
            volume >= 1000000000) {
            bulky = true;
        }
        if(mass >= 100) {
            heavy = true;
        }
        if(bulky && heavy) {
            return "Both";
        } else if (bulky) {
            return "Bulky";
        } else if (heavy) {
            return "Heavy";
        } else {
            return "Neither";
        }
    }
};